tests/test_ugrid.cpp

```diff
@@ -814,6 +814,30 @@ TEST( MeshUgridTest, flow_3d )
   MDAL_CloseMesh( mesh );
 }
 
+TEST( MeshUgridTest, IntegerTimeMesh )
```

Comment:
test coverage % dropped, see https://coveralls.io/builds/51423520/source?filename=mdal%2Ffrmts%2Fmdal_netcdf.cpp 

would it be possible to add some testcases that covers the code you have written?